<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Compressor</class>
 <widget class="QDialog" name="Compressor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>791</width>
     <height>192</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>791</width>
     <height>0</height>
    </size>
   </property>
   <property name="sortingEnabled">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Имя файла</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Разрешение</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Длительность</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Размер</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Прогресс</string>
    </property>
   </column>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>310</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labeprogressLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Прогресс сжатия</string>
   </property>
  </widget>
  <widget class="QToolButton" name="pathToolButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>260</y>
     <width>27</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pathLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить в</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="settingGroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>381</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Параметры сжатия</string>
   </property>
   <widget class="QLabel" name="qualityLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Качество сжатия</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>webm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ogv</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="formaLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Формат сжатия</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1080p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>720p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>576p</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460p</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="bitrateLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Битрейт</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>400</number>
    </property>
    <property name="maximum">
     <number>16000</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>350</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Сжать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
